dynparquet/example.go

```diff
@@ -220,3 +223,124 @@ func NewTestSamples() Samples {
 		},
 	}
 }
+
+func NewNestedSampleSchema(t *testing.T) *Schema {
+	def := &schemav2pb.Schema{
+		Root: &schemav2pb.Group{
+			Name: "nested",
+			Nodes: []*schemav2pb.Node{
+				{
+					Type: &schemav2pb.Node_Group{
+						Group: &schemav2pb.Group{
+							Name: "labels",
```

Comment:
Much the same way they do today, but instead of a column named `labels.label1` and a column named `labels.label2` they would define the struct `labels` with fields `label1` and `label2`

Reply:
No I'm not doing a good job explaining my thought well because I don't think they're fully formed. Previously we defined a labels column, and then writes would expand that column with `labels.label1` the way I had imagined it working with struct is we would only define a `labels` top-level struct and writes would define the sub fields.

Reply:
I think part of the problem was mentally I was considering removing the notion of dynamic columns. But I shouldn't try and do both here, so I'll rewrite the table test to use them and it should be more identical to what we had before